packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -337,6 +431,52 @@ See [ID Token Claims](/docs/reference/api/oidc/#id-token-claims) for a list of I
 }
 ```
 
+You can also use the `replace` operation to modify members within JSON objects and elements within arrays. For example, you have a JSON object in a claim called `employee_profile`, and you want to update the email address of the employee.
+
+The existing target JSON object:
+```json
+{
+"employee_profile": {
+       "employee_id": "1234",
+       "name": "Anna",
+       "email": "anna.v@company.com"
+    }
+}
+```
+Specify the claim in the path, followed by the name of the object member that you want to modify. 
```

Comment:
same as for https://github.com/okta/okta-developer-docs/pull/493#discussion_r296942600
and for line 448 below